patches/@microsoft__api-extractor@7.38.3.patch

```diff
@@ -1436,6 +1436,860 @@ index efd6b045915cc46c85ee3ed3e4b8a2f7104c67f9..cac9435a286e26eb79d62d473d561485
+         }
+-    }
+-    static _shouldIncludeReleaseTag(releaseTag, dtsKind) {
+-        switch (dtsKind) {
+-            case DtsRollupKind.InternalRelease:
+-                return true;
+-            case DtsRollupKind.AlphaRelease:
+-                return (releaseTag === api_extractor_model_1.ReleaseTag.Alpha ||
+-                    releaseTag === api_extractor_model_1.ReleaseTag.Beta ||
+-                    releaseTag === api_extractor_model_1.ReleaseTag.Public ||
+-                    // NOTE: If the release tag is "None", then we don't have enough information to trim it
+-                    releaseTag === api_extractor_model_1.ReleaseTag.None);
+-            case DtsRollupKind.BetaRelease:
+-                return (releaseTag === api_extractor_model_1.ReleaseTag.Beta ||
+-                    releaseTag === api_extractor_model_1.ReleaseTag.Public ||
+-                    // NOTE: If the release tag is "None", then we don't have enough information to trim it
+-                    releaseTag === api_extractor_model_1.ReleaseTag.None);
+-            case DtsRollupKind.PublicRelease:
+-                return releaseTag === api_extractor_model_1.ReleaseTag.Public || releaseTag === api_extractor_model_1.ReleaseTag.None;
+-            default:
+-                throw new Error(`${DtsRollupKind[dtsKind]} is not implemented`);
++        if (!trimmed) {
++          DtsRollupGenerator._modifySpan(
++            collector,
++            child,
++            entity,
++            childAstDeclaration,
++            dtsKind
++          );
+         }
++      }
++    }
++  }
++  static _shouldIncludeReleaseTag(releaseTag, dtsKind) {
++    switch (dtsKind) {
++      case DtsRollupKind.InternalRelease:
++        return true;
++      case DtsRollupKind.AlphaRelease:
++        return (
++          releaseTag === api_extractor_model_1.ReleaseTag.Alpha ||
++          releaseTag === api_extractor_model_1.ReleaseTag.Beta ||
++          releaseTag === api_extractor_model_1.ReleaseTag.Public ||
++          // NOTE: If the release tag is "None", then we don't have enough information to trim it
++          releaseTag === api_extractor_model_1.ReleaseTag.None
++        );
++      case DtsRollupKind.BetaRelease:
++        return (
++          releaseTag === api_extractor_model_1.ReleaseTag.Beta ||
++          releaseTag === api_extractor_model_1.ReleaseTag.Public ||
++          // NOTE: If the release tag is "None", then we don't have enough information to trim it
++          releaseTag === api_extractor_model_1.ReleaseTag.None
++        );
++      case DtsRollupKind.PublicRelease:
++        return (
++          releaseTag === api_extractor_model_1.ReleaseTag.Public ||
++          releaseTag === api_extractor_model_1.ReleaseTag.None
++        );
++      default:
++        throw new Error(`${DtsRollupKind[dtsKind]} is not implemented`);
+     }
++  }
+ }
+ exports.DtsRollupGenerator = DtsRollupGenerator;
+ //# sourceMappingURL=DtsRollupGenerator.js.map
```

Comment:
I expected the diff for this to be commenting out or deleting a couple lines. Instead, we have a ~800 line diff diff. Thats hard to review. Why is this change so big?

Reply:
Hmm. Interesting. The change _should_ only be a couple of lines. Let me try to regenerate this.

Reply:
Well, okay. The actual code change is only a couple of lines, but `pnpm patch` seems to want to include the entire file contents regardless of how I create the changes. This appears to be how the other patches were generated as well :/

Reply:
For context, though, the change is this: https://github.com/microsoft/rushstack/commit/e809553f13521a87c38ed5fa9657db793a235287

Reply:
I think your massive diff is because you reformatted the file from 2 space indent to 4 space indent, making every line changed.

Reply:
Apparently this was just a prettier issue 🤦‍♂️ Fixed now.